.github/workflows/ci-actions-incremental.yml

```diff
@@ -416,7 +419,7 @@ jobs:
           java-version: ${{ matrix.java.java-version }}
       - name: Build
         # Important: keep -pl ... in sync with "Calculate run flags"!
-        run: ./mvnw $COMMON_MAVEN_ARGS $JVM_TEST_MAVEN_ARGS clean install -pl 'integration-tests/maven'
+        run: ./mvnw $COMMON_MAVEN_ARGS $JVM_TEST_MAVEN_ARGS clean install -pl 'integration-tests/maven' -pl 'integration-tests/devmode'
```

Comment:
This job might now run devmode, although it's not affected by the respective change (e.g. only the maven module is - or vice versa), no?
So for extra efficiency, we could run GIB again here just like further up for the JVM tests job (line 330).

PS: In the end all affected/impacted modules are there already in the logfile that was created in the initial build, so we could trim that instead of running GIB again, but for now I'd go with the current solution to re-run GIB.

Reply:
@famod can you have a look at the second commit? Will GIB be triggered just by adding the additional arguments (and does it work when I actually specify a list of modules?)?

Reply:
@gsmet that should work, see also: https://github.com/quarkusio/quarkus/blob/3.3/.github/workflows/ci-actions-incremental.yml#L141-L144